frontend/awx/administration/instances/hooks/useInstanceRowActions.tsx

```diff
@@ -34,79 +43,100 @@ export function useInstanceRowActions(onComplete: (instances: Instance[]) => voi
   const userAccess = activeAwxUser?.is_superuser || activeAwxUser?.is_system_auditor;
   const isK8s = data?.IS_K8S;
 
-  return useMemo<IPageAction<Instance>[]>(
-    () => [
-      {
-        type: PageActionType.Switch,
-        ariaLabel: (isEnabled) =>
-          isEnabled ? t('Click to disable instance') : t('Click to enable instance'),
-        selection: PageActionSelection.Single,
-        isPinned: true,
-        onToggle: (instance, enabled) => handleToggleInstance(instance, enabled),
-        isSwitchOn: (instance) => instance.enabled,
-        label: t('Enabled'),
-        labelOff: t('Disabled'),
-        showPinnedLabel: false,
-        isHidden: (instance) => instance.node_type === 'hop',
-        isDisabled: (instance) =>
-          !userAccess ||
-          !isK8s ||
-          (instance?.node_type !== 'execution' && instance?.node_type !== 'hop')
-            ? t(
-                'You do not have permission to edit instances. Please contact your organization administrator if there is an issue with your access.'
-              )
+  return useMemo<IPageAction<Instance>>(
+    () => ({
+      type: PageActionType.Switch,
+      ariaLabel: (isEnabled) =>
+        isEnabled ? t('Click to disable instance') : t('Click to enable instance'),
+      selection: PageActionSelection.Single,
+      isPinned: true,
+      onToggle: (instance, enabled) => handleToggleInstance(instance, enabled),
+      isSwitchOn: (instance) => instance.enabled,
+      label: t('Enabled'),
+      labelOff: t('Disabled'),
+      showPinnedLabel: false,
+      isHidden: (instance) => instance.node_type === 'hop',
+      isDisabled: (instance) =>
+        !userAccess ||
+        !isK8s ||
+        (instance?.node_type !== 'execution' && instance?.node_type !== 'hop')
+          ? t(
+              'You do not have permission to edit instances. Please contact your organization administrator if there is an issue with your access.'
```

Comment:
There must be some difference between old app privileges check and new app checks, because Im logged as the same user in old and new app (my own user using local dev server with all privilleges) and old app allows me to change switch, but new has it disabled because of permission.

Reply:
Im still having those privilleges issues. I have user as:
![image](https://github.com/ansible/ansible-ui/assets/23277791/fe1da79a-5f2b-49f0-943e-8aa9a13181bc)

In my new application, I cant change enabled/disable and also forks, while in old, I can:
![image](https://github.com/ansible/ansible-ui/assets/23277791/b236d789-4e3e-42bc-9082-7e390a3104c5)

![image](https://github.com/ansible/ansible-ui/assets/23277791/bf6aec4c-3423-4df4-9560-8ac79e60f8d7)

Reply:
Added a commit to fix this issue should be resolved now